<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SUDOKU</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../unnamed.webp</normaloff>../../../unnamed.webp</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gl_game"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Game">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="new_game"/>
    <addaction name="open_file"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="about_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="help"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menuSolve">
    <property name="title">
     <string>Solve</string>
    </property>
    <addaction name="solve_puzzle"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionDark"/>
   </widget>
   <addaction name="menuNew_Game"/>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
   <addaction name="menuExit"/>
   <addaction name="menuSolve"/>
   <addaction name="menuMode"/>
  </widget>
  <action name="new_game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="open_file">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>Sudoku Game</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="about_2">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="solve_puzzle">
   <property name="text">
    <string>solve puzzle</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
